arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -43,6 +43,13 @@ static inline u32_t _size_to_mpu_rasr_size(u32_t size)
 		return REGION_32B;
 	}
 
+#ifdef CONFIG_SOC_SERIES_MPS2
```

Comment:
This is because of QEMU, not the actual hardware for MPS2_AN385

Reply:
Could we perhaps make this a Kconfig instead? And set to 1024 in the qemu_cortex_m3 defconfig

Reply:
How can I inform the build system that I am using qemu rather than a real hardware.
Is that I need to create a new board for this, say qemu_mps2_an385?

Reply:
no, just change qemu_cortex_m3 to use this SOC definition instead of the other one.
And then ifdef this based on BOARD=qemu_cortex_m3

Reply:
This causes build failures on all Bluetooth related tests/samples. This change would be backward incompatible.
So another solution may be something like BOARD=qemu_cortex_m3_mpu